src/lib/parsers/include/sol-json.h

```diff
@@ -435,6 +436,11 @@ int sol_json_double_to_str(const double value, char *buf, size_t buf_len);
 
 bool sol_json_is_valid_type(struct sol_json_scanner *scanner, enum sol_json_type start_type) SOL_ATTR_NONNULL(1);
 
+int sol_json_serialize_string(struct sol_buffer *buffer, const char *str) SOL_ATTR_NONNULL(1, 2);
+int sol_json_serialize_double(struct sol_buffer *buffer, double val) SOL_ATTR_NONNULL(1);
+int sol_json_serialize_int(struct sol_buffer *buffer, int val) SOL_ATTR_NONNULL(1);
```

Comment:
call it int32 and use that specific type, otherwise this will cause problems in systems where int is not 4 bytes.